frontend/public/components/events.jsx

```diff
@@ -130,7 +132,7 @@ class SysEvent extends React.Component {
       shouldAnimate = true;
     }
 
-    return <div style={style}>
+    return <div className="co-sysevent__transition" style={style}>
```

Comment:
I think this should be `co-sysevent--transition` since *transition* is modifier and we are following BEM